interfaces/apparmor/backend.go

```diff
@@ -176,6 +176,10 @@ func snapConfineFromSnapProfile(info *snap.Info) (dir, glob string, content map[
 	patchedProfileText := bytes.Replace(
 		vanillaProfileText, []byte("/usr/lib/snapd/snap-confine"), []byte(snapConfineInCore), -1)
 
+	// Replace the path to the vanilla snap-confine apparmor snippets
+	patchedProfileText = bytes.Replace(
+		patchedProfileText, []byte("/var/lib/snapd/apparmor/snap-confine"), []byte(apparmor_sandbox.SnapConfineAppArmorDir), -1)
```

Comment:
I don't see a unit test for this. Also should we ahd some checks in some spread test too?

Reply:
I added unit tests for this and the change in sandbox now

Reply:
And also added a small spread addition